### ISSUE TITLE
no-duplicates: unresolved files should use import string

### ISSUE THREAD

**Issue by benmosher** [benmosher/eslint-plugin-import#86]
```js
import 'x'
import 'y'
```
reports 'null' imported multiple times if `x` and `y` are not able to be resolved.

Need to use actual module string for unresolved modules.
